File: app/src/main/java/com/sproutling/object/PushNotificationData.java
/*
 * Copyright (C) 2017 Mattel, Inc. All rights reserved.
 */

package com.sproutling.object;

import com.google.gson.annotations.SerializedName;

/**
 * Created by subram13 on 4/10/17.
 */

public class PushNotificationData {
    @SerializedName("title")
    private String mTitle;
    @SerializedName("body")
    private String mBody;

    public PushNotificationData(String title, String body) {
        mTitle = title;
        mBody = body;
    }

    public String getTitle() {
        return mTitle;
    }

    public String getBody() {
        return mBody;
    }
}

File: common/src/main/java/com/sproutling/common/pojos/events/PushNotificationData.kt
/*
 * Copyright (C) 2017 Mattel, Inc. All rights reserved.
 */

package com.sproutling.common.pojos.events

import com.google.gson.annotations.SerializedName

/**
 * Created by subram13 on 4/10/17.
 */

class PushNotificationData(@field:SerializedName("title")
                           val title: String, @field:SerializedName("body")
                           val body: String)
